rpcs3/Emu/Io/Buzz.h

```diff
@@ -8,9 +8,13 @@ class usb_device_buzz : public usb_device_emulated
 	usb_device_buzz(u32 first_controller, u32 last_controller, const std::array<u8, 7>& location);
 	~usb_device_buzz();
 
+	static std::shared_ptr<usb_device> make_instance(u32 controller_index, const std::array<u8, 7>& location);
+
 	void control_transfer(u8 bmRequestType, u8 bRequest, u16 wValue, u16 wIndex, u16 wLength, u32 buf_size, u8* buf, UsbTransfer* transfer) override;
 	void interrupt_transfer(u32 buf_size, u8* buf, u32 endpoint, UsbTransfer* transfer) override;
 
+	static u16 get_num_emu_devices();
```

Comment:
is there a reason why you provide in all `.h` this static method here and not after the other static method while in the `.cpp` they are one after another?

Reply:
Nope, I'll fix that. Do we have a preference for order when it comes to static methods like this?

Reply:
have moved them all to after the other static method now